front/src/modules/pathfinding/hook/usePathfinding.ts

```diff
@@ -285,6 +285,7 @@ export const usePathfindingV2 = (
               );
 
               setPathProperties?.({
+                manchetteOperationalPoints: operational_points,
```

Comment:
I think we should not update the manchetteOperationalPoints here, it is used only in the results so it should be set directly there (in `useStdcmResults.ts`)